janitor/janitor.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM janx/ubuntu-dev
+MAINTAINER Jan Keromnes "janx@linux.com"
+
+# Download Janitor's source code.
+RUN git clone https://github.com/jankeromnes/janitor
```

Comment:
Also needs `--recursive` to checkout the `dockerfiles` Git submodule, and a `npm update` is also required to install all dependencies.